### PR TITLE
Cleanup for rank commands

### PR DESCRIPTION
**Pull request by mathiascode**
Combined the unrank and deop commands, since they're pretty much the same.

### CODE REVIEW

File: cmd_rank.lua [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	end
+
+	if not PlayerName then
+		Response = SendMessage(Player, "Usage: " .. Split[1] .. " <player> [rank]")
+	else
+		-- Translate the PlayerName to a UUID:
+		local PlayerUUID = GetPlayerUUID(PlayerName)
+
+		if not PlayerUUID or string.len(PlayerUUID) ~= 32 then
+			return true, SendMessage(Player, "There is no player with the name \"" .. PlayerName .. "\"")
```
Comment (bearbin) [inline cmd_rank.lua]: Why break the pattern here, I don't think the benefit from making this a guard clause overrides the confusion.

File: cmd_rank.lua [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	local Response
+	local PlayerName = Split[2]
+	local NewRank = Split[3]
+	
+	local InformLoadRank = function(OtherPlayer)
+		if PlayerName == OtherPlayer:GetName() then
+			local AuthorName = "the server console"
+			
+			if Player then
+				AuthorName = "\"" .. Player:GetName() .. "\""
```
Comment (bearbin) [inline cmd_rank.lua]: Do we need the quotation marks?

File: cmd_unrank.lua [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+		
+		if Player then
+			AuthorName = "\"" .. Player:GetName() .. "\""
+		end
+
+		OtherPlayer:SendMessageInfo("You were unranked by " .. AuthorName)
+		OtherPlayer:LoadRank()
+	end
+
+	if not Split[2] then
```
Comment (bearbin) [inline cmd_unrank.lua]:
```suggestion
	if not PlayerName then
```

File: cmd_rank.lua (hunk 1 above)
Comment (mathiascode) [reply cmd_rank.lua]: Logic flies out the window sometimes when you stare at code for too long...

File: cmd_rank.lua (hunk 2 above)
Comment (mathiascode) [reply cmd_rank.lua]: I personally prefer them, especially if the player has a username that blends in with the sentence.

Comment (bearbin) [reply cmd_rank.lua]: I don't really mind, but we should be consistent one way or the other. I see that in your other cleanup that we have the quotation marks so they're fine.

**Pull request merged**